x-pack/platform/plugins/shared/fleet/server/types/models/agent_policy.ts

```diff
@@ -158,6 +158,18 @@ export const AgentPolicyBaseSchema = {
       ),
     })
   ),
+  fleet_ui: schema.maybe(
```

Comment:
I am confused with the `fleet_ui` part here, there are a lot of other agent policies properties that are not part of the `.fleet-policies` document do we plan to migrate all of them to `fleet_ui`?  if not it seems this could go to the root level no?

Reply:
I think you are right, we can just put `required_versions` under the root level, as it doesn't automatically show up in the deployed policy.

Reply:
updated